src/packages/core/workspace/workspace-property-layout/workspace-property-layout.element.ts

```diff
@@ -80,18 +86,28 @@ export class UmbWorkspacePropertyLayoutElement extends LitElement {
 				grid-column: span 2;
 			}
 
+			:host > div:last-child {
+				margin-top:var(--uui-size-space-3);
+			}
+
 			@container (width > 600px) {
 				:host(:not([orientation='vertical'])) > div {
 					grid-column: span 1;
+					margin-top:0;
 				}
 			}
 
 			:host(:last-of-type) {
 				border-bottom: none;
 			}
 
+			/* container has 18px padding, reduce here to ensure consistency */
 			:host-context(umb-variantable-property:first-of-type) {
```

Comment:
I can see this :host-context was here already, but I would not recommend using such.

With the perspective of separation of concerns, then we do not want web-components to be aware about other web-components. Especially not linking them by element name.

Looking at the problem I would say its a implementation problem within `umb-document-workspace-view-edit-tab`.

Here we have the ability to overwrite the padding of uui-box, via an CSS Custom Prop. in this way:

`
uui-box {
	--uui-box-default-padding: 0 var(--uui-size-space-5);
}
`